packages/flutter_tools/bin/hack_script.sh

```diff
@@ -0,0 +1,7 @@
+eval "$(echo "static const int Moo = 88;" | xcrun clang -x c \
```

Comment:
I feel like something named `hack_script.sh` needs a comment providing some context (and a bug link, ideally)

Reply:
There was no reason for me to leave this in. I've cleaned it up so that this is no longer a bash scrip.